fsspec/implementations/local.py

```diff
@@ -20,6 +20,7 @@ class LocalFileSystem(AbstractFileSystem):
     """
 
     root_marker = "/"
+    _add_aliases = True
```

Comment:
Could just be the default on AbstractFileSystem, not repeated for every derivative? 
Could simply remove and add aliases regardless?

Reply:
No. This is the only way to know that a subclass as *opted in* to having these aliases. Otherwise, we wouldn't be able to know if we need to warn or not.

Reply:
I am thinking we might just making mandatory and skip opting out or in, and no need for warnings. That would be equivalent to simply including them in spec

```
    cp = copy
```
etc

Reply:
Fine by me. Do you recall the motivation for not doing that in the first place?

Reply:
Not as such. It is possible (should be checked) that the overrides in gcsfs and s3fs do not agree on which one of a pair of possibilities they implement.
Actually, thinking: if `cp = copy` in the superclass and the subclass defines its `copy`, does it get the subclass `cp`?

Reply:
No, it does not :|

Reply:
I don't follow. Are you talking about master? Things seem OK on this branch.

```
In [2]: import fsspec

In [3]: class MyFS(fsspec.AbstractFileSystem):
   ...:     def copy(self, *args, **kwargs):
   ...:         print("my copy!")
   ...:         return super().copy(*args, **kwargs)
   ...:

In [4]: fs = MyFS()

In [6]: fs.cp('a', 'b')
my copy!
```

Reply:
I'm talking about the putative really simple way to do it, which it turns out doesn't work. Your version works correctly, so I'm happy to go with it